physics/mynnedmf_wrapper.F90

```diff
@@ -140,16 +135,18 @@ SUBROUTINE mynnedmf_wrapper_run(        &
      &  nupdraft,maxMF,ktop_plume,      &
      &  dudt, dvdt, dtdt,                                  &
      &  dqdt_water_vapor,            dqdt_liquid_cloud,    & ! <=== ntqv, ntcw
-     &  dqdt_ice_cloud,              dqdt_ozone,           & ! <=== ntiw, ntoz
+     &  dqdt_ice_cloud,              dqdt_snow_cloud,      & ! <=== ntiw, ntsw
```

Comment:
We should have no say about what variables are called locally, but I'm wondering why this is `dqdt_snow_cloud` instead of just `dqdt_snow`. It's referring to the tendency of the snow tracer which presumably is grid mean and sediments out of the cloud.

Reply:
Yes, I just have been in a few discussions recently about "what is a cloud?" If snow settles beneath the "snow cloud", when does it become a cloud in the k-1 layer? When do non-zero cloud fractions become appropriate? From a modeling point of view, it really depends on the cloud fraction parameterization, but which one is correct?

Well, I prefer shorter names, so I don't mind removing "cloud" from the name.